src/_DataStructures_/BloomFilters/index.js

```diff
@@ -0,0 +1,97 @@
+class BloomFilters {
+  constructor(size = 101) {
+    this.size = size;
+    this.data = this.getStorage(size);
+  }
+
+  add(element) {
+    const indices = this.getIndices(element);
+
+    for (let i = 0; i < indices.length; i += 1) {
+      this.data.setBit(indices[i]);
+    }
+  }
+
+  contains(element) {
+    const indices = this.getIndices(element);
+
+    for (let i = 0; i < indices.length; i += 1) {
+      const index = indices[i];
+      if (!this.data.getBit(index)) {
+        return false; // item is definately not there
+      }
+    }
+    return true; // item may be there
+  }
+
+  getIndices(element) {
+    return [this.hashOne(element), this.hashTwo(element), this.hashThree(element)];
+  }
+
+  hashOne(value) {
+    const stringValue = String(value);
+    let hashVal = 0;
+
+    for (let i = 0; i < stringValue.length; i += 1) {
+      hashVal += stringValue.charCodeAt(i) - 96;
+    }
+
+    // eslint-disable-next-line no-bitwise
+    hashVal &= hashVal;
```

Comment:
Why this operation is required?

Reply:
To get a better hash. It may look useless but here is the explanation: https://stackoverflow.com/questions/38356644/why-is-the-bitwise-and-of-two-of-the-same-value-producing-a-different-value

I must update it in the comments as well